src/seedu/addressbook/parser/Parser.java

```diff
@@ -96,7 +96,7 @@ public Command parseCommand(String userInput) {
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareAdd(String args){
+    private static Command prepareAdd(String args){
```

Comment:
Good job on making these private methods static as well. As `parseCommand` was made static, any methods that it depends on has to be made static as well.